cms/models/pluginmodel.py

```diff
@@ -21,7 +19,18 @@
 from cms.utils.conf import get_cms_setting
 from cms.utils.urlutils import admin_reverse
 
-from treebeard.mp_tree import MP_Node
+
```

Comment:
raw sql? what dialects are supported?

Reply:
We'll support Postgres for now.
Next pr will add sqlite and mysql support